MAVProxy/modules/mavproxy_rc.py

```diff
@@ -112,10 +112,10 @@ def mavlink_packet(self, m):
 
     def send_rc(self):
         '''send RC override packet'''
-        if self.sitl_output:
+        if self.sitl_outputs:
             chan16 = self.override[:16]
             buf = struct.pack('<HHHHHHHHHHHHHHHH', *chan16)
-            self.sitl_output.write(buf)
+            self.sitl_outputs[self.master.linknum].write(buf)
```

Comment:
This looks like it is conflating link numbers and sitl-output-port numbers.

That probably means you could elicit a runtime exception with `link add` - or simply starting with a different number of links to sitl-rc-outputs.

There isn't a 1:1 correspondence between link numbers and SITL outputs - you could have a connection to the same simulation process on 2 of its mavliunk ports, for example.